e2e/app/google-login.ts

```diff
@@ -89,8 +89,13 @@ export default class GoogleLoginPage extends BasePage {
   /**
    * Open All-of-Us Google login page.
    */
-  async goto(): Promise<void> {
-    await this.page.goto(configs.uiBaseUrl + configs.loginUrlPath, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 0});
+  async load(): Promise<void> {
+    const url = configs.uiBaseUrl + configs.loginUrlPath;
+    await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000}).catch((err) => {
+      console.error('Google login page not found. ' + err);
+      this.takeScreenshot('GoogleLoginNotFound');
```

Comment:
May not be part of this PR but it will be nice to have a util method to handle errors where in we can just pass the flow name say invitation key -> it will print error and take a screen shot with file name invitation key

Reply:
I'll make a note of this, for next PR.